index.html

```diff
@@ -232,6 +232,53 @@ <h2>About me</h2>
       </div>
     </div>
   </section>
+  <footer>
+    <div class="footer-section">
+    <div>
+      <p class="footer-p">
+        I'm always interested in hearing about new projects,
+         so if you'd like to chat please get in touch.
+      </p>
+    </div>
+   <div>
+    <form action="https://formspree.io/f/mwkzggnv" method="post">
+      <input type="text" placeholder="Full name" maxlength="30" id="text" name="text" required>
+      <input type="email" id="email" name="email" placeholder="Email address" maxlength="30" required>
+      <textarea name="" id="" cols="10" rows="5" required maxlength="500" name="textarea" id="textarea"  placeholder="Enter text here"></textarea>
+      <button class="Multi-Post-project button form-button" type="submit">Get in touch</button>
+    </form>
```

Comment:
- [ ] Please use flexbox to lay out elements in the form. This is a project requirement. 🤝 

![ins](https://user-images.githubusercontent.com/99656687/220324408-45f6fd9d-6bec-467b-9d42-2bf4f0b5957c.png)

Reply:
- [ ] Please implement the button states on hover and active as required in the project requirements 👍🏾 
![btns](https://user-images.githubusercontent.com/99656687/220324873-2b763e0f-3538-4ca1-a6c0-7aa7bb61769d.png)